binding/react/src/use_rhino.ts

```diff
@@ -176,6 +183,7 @@ export function useRhino(
       const {
         accessKey,
         context,
+        endpointDurationSec,
         requireEndpoint,
```

Comment:
defaults?

Reply:
We have some default values for these in the web bindings, and since this is only a wrapper around them, no default is set here.